<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Psy\Test\Exception\RuntimeExceptionTest" namespace="Psy\Test\Exception" name="RuntimeExceptionTest" abstract="false" final="false" start="17" end="31">
  <file path="/home/web/example/src/freegame_developer/vendor/psy/psysh/test/Psy/Test/Exception" file="RuntimeExceptionTest.php" realpath="/home/web/example/src/freegame_developer/vendor/psy/psysh/test/Psy/Test/Exception/RuntimeExceptionTest.php" size="778" time="2016-03-09T05:03:14+00:00" unixtime="1457499794" sha1="696501da9c995f2e2fa7dec1efb5196e560ea984" relative="freegame_developer/vendor/psy/psysh/test/Psy/Test/Exception/RuntimeExceptionTest.php"/>
  <extends full="PHPUnit_Framework_TestCase" namespace="" name="PHPUnit_Framework_TestCase"/>
  <method name="testException" start="19" end="30" abstract="false" final="false" static="false" visibility="public"/>
</class>
